packages/complex-matcher/index.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
-const { escapeRegExp, isPlainObject, some } = require('lodash')
+const escapeRegExp = require('lodash/escapeRegExp.js')
+const isPlainObject = require('lodash/isPlainObject.js')
+const some = require('lodash/some.js')
```

Comment:
I don't know if this is important, but the majority of `require` on this repo seem not to use the `.js` extension.

```suggestion
const escapeRegExp = require('lodash/escapeRegExp')
const isPlainObject = require('lodash/isPlainObject')
const some = require('lodash/some')
```

Reply:
CommonJS accepts both but ECMAScript modules require the extension.

Because we are moving forward to ESM, I'm now always using them :slightly_smiling_face: